app/templates/_base_page.html

```diff
@@ -63,6 +63,15 @@
       {% endfor %}
     {% endwith %}
   {% endblock flashMessages %}
+  {% block errorSummary %}
+    {% if errors %}
+      {{ govukErrorSummary({
+      "titleText": error_title or "There is a problem",
+      "descriptionText": error_description or None,
```

Comment:
I think we need to use `descriptionHtml`, for when there is an embedded link

Reply:
That's what I thought too at first, but wrapping the text with a link in `Markup()` seems to work fine: 

![image](https://user-images.githubusercontent.com/6362602/108074818-87f43800-7061-11eb-93d2-d3f287b8486d.png)